### DISCUSSION TITLE
Can't reverify university

### DISCUSSION THREAD

**Discussion by thiagorbernardo** [community/community#49660] [GitHub Education] [answered]
I'm trying to renew my student application, but when I select my school email I receive the following prompt:
`*****@alunos.utfpr.edu.br has already been used to obtain an academic discount on GitHub account(s) thiagorbernardo.`
<img width="765" alt="Screenshot 2023-03-09 at 22 17 56" src="https://user-images.githubusercontent.com/51447939/224198620-0e484341-3bcc-4109-87a3-e93b5c86d23e.png">

The problem is, this thiagorbernardo is me, so why this error?

If I ignore this error and try to apply I receive this other error:

`We require applicants to use a school-issued email address to apply.`
![Second Prompt and Proof](https://user-images.githubusercontent.com/51447939/224198167-024f2013-9ef1-4568-bc76-5308b8cf5c96.png)

And again, this is a school email that I was using on GitHub since 2021. (Proof on the print above saying that I'm on the program since 2021.

Please, if you can help me.

Thank you.

**Comment by samus-aran** [comment, marked as answer]
Hi @thiagorbernardo 

We have updated these school's details. You should be able to select your campus and use that email domain. Please try and re-apply 🙂 

Let us know how you get on 👍

**Reply by thiagorbernardo** [reply]
I sent the application, thank you!!!
